target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -782,7 +855,8 @@ TARGET_DEVICES += CVG834G
 
 define Device/DG834GT_PN
   $(Device/bcm63xx)
-  DEVICE_TITLE := Netgear DG834GT/PN
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_MODEL := DG834GT/PN
```

Comment:
Same question as above: One name or two names?

Reply:
These seem to be two devices: https://en.wikipedia.org/wiki/Netgear_DG834_(series)

Reply:
Sounds logical to me. So let's split them, too.